client/src/client/database_manager_client.py

```diff
@@ -29,7 +29,7 @@ def open(self, passphrase: str = "") -> Optional[str]:
             db_proxy.initialize(self.db)
             self.db.create_tables([Friend, Message, Setting])
         except peewee.DatabaseError as e:
-            if str(e) == "file is encrypted or is not a database":
+            if str(e) == "file is not a database":
```

Comment:
As noted above: assume the password is correct since it decrypted the keys. Any exception should be propagated upwards (maybe wrapped in a custom exception)

Reply:
I moved all exception checking to startup sequence.